docs/developer/procedures/contribution_license.rst

```diff
@@ -5,10 +5,10 @@ GeoTools has adopted a formal policy as part of the process of joining the Open
 
 All new :doc:`contributors </roles/contributor>` will be required to grant copyright to the foundation using a `Contributor Licenses <https://www.osgeo.org/about/licenses/>`_:
 
-* individual_contributor (:download:`txt </artifacts/individual_contributor.txt>` | :download:`pdf </artifacts/individual_contributor.pdf>`)
-* corporate_contributor (:download:`txt </artifacts/corporate_contributor.txt>` | :download:`pdf </artifacts/corporate_contributor.pdf>`)
+* `individual_contributor <https://www.osgeo.org/resources/individual-contributor-license/>`__
+* `corporate_contributor <https://www.osgeo.org/resources/corporate-contributor-license/>`__
 
-These licenses are directly derived from the Apache code contribution licenses (CLA V2.0 and CCLA v r190612).
+GeoTools is grateful that organizations of all shapes and sizes support our project with in-kind participation of their employees. Extending commit access is made to individuals directly based on their expertise demonstrated over time.
```

Comment:
The line above about apache is vital to communicate.  For a while we had our own homemade contributor license and no large corporations could communicate due to lack of legal evaluation. google eventually asked us to change so their employees could contribute back. 

apache license is well understood legally.